python/ray/autoscaler/cli_logger.py

```diff
@@ -0,0 +1,426 @@
+import sys
+
+import click
+
+import colorama
+from colorful.core import ColorfulString
+import colorful as cf
+colorama.init()
+
+def _tostr(x):
+    if not isinstance(x, str):
+        return str(x)
+    return x
```

Comment:
isn't this the same as calling `str(x)`?

Reply:
oh, apparently python does this differently, I thought that `str("a") == ""a""` but it actually does not add extra quotes